modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/layer/SymbolMBLayer.java

```diff
@@ -1731,7 +1731,12 @@ public Expression textTranslateAnchor() {
         // layer.textHaloBlur();
 
         // auto wrap
-        // symbolizer.getOptions().put("autoWrap", layer.textMaxWidth()); //  Pixels (GS) vs ems (MB); Vendor options with expressions?
+        // uses getTextSize to convert text-max-width from ems to pixels
+        if(getTextMaxWidth() != null) {
+            symbolizer.getOptions().put("autoWrap", String.valueOf(getTextMaxWidth().intValue() * getTextSize().intValue())); 
+        } else{
+            symbolizer.getOptions().put("autoWrap", String.valueOf(getTextSize().intValue() * 10));
```

Comment:
If 10 is the MapBox default, just add a comment mentioning that and leave it as-is then.

(StyleFactory would just give us the SLD default - best to stick to the MapBox default where it exists)